SSO: Remove tier, recommend GitHub first (#1255)
semgrep/semgrep-docs: docs/semgrep-cloud-platform/sso.md

#  Single-sign on (SSO) configuration

Semgrep Cloud Platform supports SSO through [OpenID Connect / OAuth2](#openid-connect--oauth2) and [SAML 2.0](#saml-20).

:::warning

> If you are planning to connect your Semgrep organization to GitHub organization, follow the steps at [Creating a Semgrep organization account](/docs/semgrep-cloud-platform/user-management/#creating-a-semgrep-organization-account) first to connect to GitHub. You can add SSO later.

Reviewer: ```suggestion
If you are planning to connect your Semgrep organization to your GitHub organization, follow the steps at [Creating a Semgrep organization account](/docs/semgrep-cloud-platform/user-management/#creating-a-semgrep-organization-account) first to connect to GitHub. Then, add SSO.
```
